mentors.html

```diff
@@ -0,0 +1,149 @@
+---
+---
```

Comment:
As I mentioned in previous comment, a lot of issues are related to size and also the page is very long. So, to solve this I propose this change, where we will just list the mentors students and contributors on the main page and use a separate html for Full version with cards to cut the size.